torchio/utils.py

```diff
@@ -206,3 +206,19 @@ def sitk_to_nib(image: sitk.Image) -> Tuple[np.ndarray, np.ndarray]:
     affine[:3, :3] = RZS
     affine[:3, 3] = translation
     return data, affine
+
+
+def exact_round(value):
```

Comment:
Can you rename this to `round_up` and add typing hints? I.e. `def round_up(value: float) -> float:` and explain that it rounds toward infinity? This method is not really more exact, it's just a different convention to Python's.

Reply:
Done